.gitignore

```diff
@@ -319,6 +319,7 @@ bh_unicode_properties.cache
 GitHub.sublime-settings
 
 ### VisualStudioCode ###
+.vscode
```

Comment:
I'm actually surprised we make these exceptions. I'd like to know more of what is the use case here.
I started using a vscode extension that colors the windows of each open project differently to help when working with multiple open windows. However, the extension adds the file and some settings to .vscode/settings.json that would be irrelevant to any other dev working on the same repo. I could see a potential in some cases, but we don't all work with vscode or have any work processes that revolve around vscode and a specific set of extensions or settings. 

Since adding the extension, all the repos I work on want to add this file, so I've been adding the .vscode line so they don't get committed. Alternatively I could also remove the `!.vscode/settings.json` line - it would achieve the same.

Maybe it's better to remove all of those exceptions?

Reply:
I've done a little more digging on this, and IMO, I think we should remove the _exceptions_ in our .gitignore. I've been playing around with creating a VSCode extension (for viewing/validating dot org readme files) in my free time, and now looking at that project these are the kinds of files included in the folder, and seem to be needed if you're going to commit the extension code. In my extension I'm using `launch.json` and `extension.json`, specific for the extension's functionality. It's also using `settings.json`, but only for the color customizations I mentioned above.

What do you think?

Reply:
I've gone ahead and removed the exceptions, but happy to add them back if you think they're needed.